manifests/image-references

```diff
@@ -2,7 +2,7 @@ kind: ImageStream
 apiVersion: image.openshift.io/v1
 spec:
   tags:
-  - name: service-serving-cert-signer
+  - name: openshift-service-cert-signer-operator
```

Comment:
The name here needs to match the prefix you have when built with CI, so this was previously correct.

However, your link below is wrong, it should be whatever you have in your file:

```
name: quay.io/openshift/origin-service-serving-cert-signer
```

has to match

```
imagePullSpec: quay.io/openshift/origin-service-serving-cert-signer:latest
```